simple-csweb.user.css

```
@@ -8,6 +8,100 @@
 @var checkbox show-right-sidebar "Show Right Sidebar" 0
 ==/UserStyle== */
 
+@-moz-document regexp("https://csweb.u-aizu.ac.jp/campusweb/campusportal.do(\?locale.*|)") {
+  #portal-body {
+    background-color: #f7f7f7;
+    height: 100vh;
```

```suggestion
    height: 85vh;
```
ちょっと上げてもかまいませんか?(参考: https://bulan.co/swings/the_illusion_in_the_design/)